BiweeklyMeetings/2022-05-24.md

```diff
@@ -0,0 +1,575 @@
+
+**M. Àngels Jover**:  Looks like it sounds better now, right? Or no?
+
+**Robert Phair**: Someone just did something. Someone just did something a few seconds ago, then it was... I've been clicking on my unmute button and I've been told... I've been suppressed ever since I reconnected. So I guess, whatever you did, fixed it. I've been trying like hell just to unsuppress myself and I couldn't. So thank you. Yeah, that's it.
+
+**Robert Phair**: All the comments that I wanted to make so far are in the chat. So I don't see Maria that... Even in the chat rooms. So that's where I've commented, is just in the chat. I don't know if I can be inserted into the minutes somehow, but there were a couple of things, but... What would the process be for in...
+
+**Robert Phair**: Just a comment, if there were relevant things in the chat to the discussion, are they ever inserted into the minutes or does it have to be spoken?
+
+**M. Àngels Jover**:  I had a little bit of trouble understanding.
+
+**Matthias Benkort**: [inaudible 00:44:48] to the chat. So Robert was asking, what do we do with the chat basically, regarding the minutes meeting? I think we should just include them in some form or another, because there has been relevant stuff being said in the chat, and we don't bring everyone on vocal anyway. So, that's important.
+
+**Robert Phair**: Okay. Well, you can edit what I've said, because a lot of it is, "Help, I'm being suppressed." So you can turn that down a little bit. I did make some comments about two of the other proposals, but that's enough. We can keep moving on. Are you still hearing any noise that you were noticing through the web app?
+
+**M. Àngels Jover**:  It's still noisy and it's difficult to understand you, because it gets... I mean the words... Well, nevermind. It is difficult.
+
+**Robert Phair**: [inaudible 00:45:42]? Okay.
+
+**M. Àngels Jover**:  What I will do is check and see how we can include this, because I think that what you mentioned is really interesting. So I will check how to include the comments on the meeting notes.
+
+**Robert Phair**: Is this a better option than going down? Does that make a difference at all?
+
+**M. Àngels Jover**:  Not really.
+
+**Matthias Benkort**: Slightly, but it's still not the best.
+
+**Robert Phair**: Okay. This is probably the best I can do. So I'll give a little background largely within the figures, but obviously that's... Maria doesn't think that's as bad. This is it. I think we got... It's maybe a different process... Audio processing than Crowdcast. I'll try to test it. If anyone wants to call me privately, I'll be able to test it with them. But until then, there's nothing I can do. So I'll mute myself again, and just insert the text comments when I have something to say.
+
+**M. Àngels Jover**:  No problem. If you want Robert, we can later, after this meeting or maybe another day, we can do some testing together.
+
+**Robert Phair**: Sure. Yeah, anytime, just let me know.
```

Comment:
```suggestion
```
The dialogue was resolved before this section & only incidental to the mic problem & not the meeting itself.  Thanks again @mangelsjover for taking the time after the meeting to test my audio.